src/project/manifest/mod.rs

```diff
@@ -355,6 +355,14 @@ impl Manifest {
             miette::bail!("pixi does not support wildcard dependencies")
         };
 
+        // Check for duplicates.
+        if dependency_table.contains_key(name.as_source()) {
```

Comment:
You want the normalized version

Reply:
It is inserted into the table as source:

```rs
dependency_table.insert(name.as_source(), Item::Value(spec.to_string().into()));
```

Reply:
Yeah so either insert as normalized or make the check normalized. I would say insert as normalized. Not sure why you want to add it as source